PaulasWeatherApp/PaulasWeatherApp/Model/Weather.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  Weather.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 7/26/22.
+//
+
+import Foundation
+
+struct WeatherDataModel: Codable {
+    
+    let lat, lon: Double
+    let timezone: String
+    let timezoneOffset: Int
+    let current: Current
+    let hourly: [Current]
+    let daily: [Daily]
+    
+    enum CodingKeys: String, CodingKey {
+        case lat, lon, timezone
+        case timezoneOffset = "timezone_offset"
+        case current, hourly, daily
+    }
+}
+
+struct Current: Codable {
```

Comment:
"Current" doesn't say a lot about this structure. It could be renamed to something like WeatherDetails